### PR TITLE
Admin can delete tutorial

### LINKED ISSUE #5: Delete videos when deleting a Tutorial
Background: Currently deleting a Tutorial will leave all of the videos in the database, meaning they will be referencing a tutorial that doesn't exist.

Use ActiveRecords dependent destroy functionality to fix this.

### PR DESCRIPTION
**Pull request by hsmitha26**
closes #5 
All tests passing.
Ran Rubocop, still have 8 offenses.
No new routes were added.

Adds the following:
- new test file spec/features/admin/admin_can_delete_tutorial_spec.rb to test that an admin can delete tutorial and its associated videos.
- dependent: :destroy to Tutorial model.
- changes route when admin clicks on Destroy to admin_tutorial_path(tutorial) from tutorial_path(tutorial)
- creates destroy action in admin/tutorials_controller.rb

### CODE REVIEW

File: app/controllers/admin/tutorials_controller.rb [hunk 1]
```diff
@@ -29,6 +29,13 @@ def update
     redirect_to edit_admin_tutorial_path(tutorial)
   end
 
+  def destroy
+    tutorial = Tutorial.find(params[:id])
+    tutorial.destroy
+    redirect_to admin_dashboard_path
+    # binding.pry
```
Comment (tschaffer1618) [inline app/controllers/admin/tutorials_controller.rb:36]: Looks good! This binding.pry can be removed at any time.

File: app/models/tutorial.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Tutorial < ApplicationRecord
-  has_many :videos, -> { order(position: :ASC) }
+  has_many :videos, -> { order(position: :ASC) }, dependent: :destroy
```
Comment (tschaffer1618) [inline app/models/tutorial.rb:4]: Good job destroying the dependencies as well.

File: spec/features/admin/admin_can_delete_tutorial_spec.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    within(first('.admin-tutorial-card')) do
+      click_on 'Destroy'
+    end
+
+    expect(current_path).to eq(admin_dashboard_path)
+    expect(page).to_not have_content(t1.title)
+    expect(page).to have_content(t2.title)
+    expect(page).to have_content(t3.title)
+  end
+end
```
Comment (tschaffer1618) [inline spec/features/admin/admin_can_delete_tutorial_spec.rb:41]: I like the test for each case (with videos and without) here. Very thorough.

**Review by tschaffer1618 — APPROVED** [review]

File: app/controllers/admin/tutorials_controller.rb (hunk 1 above)
Comment (hsmitha26) [reply app/controllers/admin/tutorials_controller.rb:36]: Oops! Thanks for catching that.  I should starting searching the whole for prys before pushing the branch.

**Pull request merged**